test/runtime/Policies.go

```diff
@@ -669,6 +671,239 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		connectivityTest(allRequests, helpers.App2, helpers.Httpd1, BeTrue)
 	})
 
+	It("Checks CIDR Policy", func() {
+
+		ipv4Host := "192.168.254.254"
+		ipv4OtherHost := "192.168.254.111"
+		ipv4OtherNet := "99.11.0.0/16"
+		ipv6Host := "fdff::ff"
+		httpd2Label := "id.httpd2"
+		httpd1Label := "id.httpd1"
+
+		// Helper function which creates a file containing policyText which is
+		// assumed to be a valid Cilium Network Policy, imports it, and waits for
+		// the policy to be realized within Cilium. Deletes the created file when
+		// done.
+		renderAndImportPolicy := func(policyText string) {
```

Comment:
Not sure why you use this function, but there is a helper for this: `vm.PolicyRenderAndImport(string)`

Reply:
Done. Removed this function and replaced all uses with `vm.PolicyRenderAndImport`.